### PR TITLE
Switch to lib

### PR DESCRIPTION
**Pull request by stephanieea**
This moves our main and renderer to the lib folder. 🎉 
Closes Issue #10 

It also includes the base for the timer- which I will adjust to increment up by minutes per our class discussion .

How do I include gifs?  @apsitos @gprocell927

### CODE REVIEW

File: lib/renderer.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+// Helper function, to format the time
+const secondsToTime = (s) => {
+  const momentTime = moment.duration(s, 'seconds');
+  const sec = momentTime.seconds() < 10 ? ('0' + momentTime.seconds()) : momentTime.seconds();
+  const min = momentTime.minutes() < 10 ? ('0' + momentTime.minutes()) : momentTime.minutes();
+  const hours = momentTime.hours() < 10 ? ('0' + momentTime.hours()) : momentTime.hours();
+  return `${hours}:${min}:${sec}`;
+}
+
```
Comment (apsitos) [inline lib/renderer.js:40]: This looks great! In our next iteration, let's only increment the minutes? ⏲

Comment (stephanieea) [reply lib/renderer.js:40]: For sure!!!

Comment (stephanieea) [reply lib/renderer.js:40]: 🇹🇱

File: style.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.timer {
+    font-size: 60px;
+    background: #0B0D0F;
```
Comment (gprocell927) [inline style.css:3]: What about blanched almond? :trollface:

Comment (apsitos) [reply style.css:3]: Or Peru? 🇵🇪

**Comment by apsitos** [comment]
Fantastic! We have a working timer! ⌛️

**Pull request merged**